frontend/app/views/fragments/giraffe/abFormTrack.scala.html

```diff
@@ -0,0 +1,9 @@
+@import views.support.ChosenVariants
+@import java.net.URLEncoder
+@(chosenVariants: ChosenVariants)
+
+
+    <input type="hidden" name="abTest" class="js-ab-test" value="@URLEncoder.encode(chosenVariants.asJson,"UTF8")">
```

Comment:
You could put 

URLEncoder.encode(chosenVariants.asJson,"UTF8")

as a function on chosenVariants for simplicity